extensions/2.0/Khronos/KHR_materials_diffuse_transmission/README.md

```diff
@@ -0,0 +1,404 @@
+    <td><img src="figures/color_1.0.jpg"/></td>
+  </tr>
+ <tr>
+    <td align="center">0.0</td>
+    <td align="center">0.25</td>
+    <td align="center">0.5</td>
+    <td align="center">0.75</td>
+    <td align="center">1.0</td>
+  </tr>
+  <tr>
+    <td colspan="5" align="center">
+      <em>Backlit, occluded plane with blue <code>baseColorFactor</code> and red <code>diffuseTransmissionColorFactor</code> for varying <code>diffuseTransmissionFactor</code>.</em>
+    </td>
+  </tr>
+</table>
+
+### diffuseTransmissionTexture
+
+The `A` channel of this texture defines proportion of light that is diffusely transmitted through a surface, rather than being diffusely re-emitted. This is expressed as a percentage of the light that penetrates the surface (i.e., not specularly reflected), rather than a percentage of the total light incident on the surface. A value of 1.0 indicates that 100% of the light that penetrates the surface is transmitted through it.
+
+The value is linear and is multiplied by the `diffuseTransmissionFactor` to determine the total diffuse transmission value.
+
+```
+diffuseTransmission = diffuseTransmissionFactor * diffuseTransmissionTexture.a
+```
+
+<table>
+ <tr>
+    <td><img src="./figures/factor_tex_inlay.jpg"/></td>
+  </tr>
+  <tr>
+    <td align="center">
+      <em>Backlit, occluded plane with blue <code>baseColorFactor</code> and a striped <code>diffuseTransmissionTexture</code>.<br>(Input texture shown in the top-left).</em>
+    </td>
+  </tr>
+</table>
+
+
+### diffuseTransmissionColorTexture
+
+The `RGB` channels of this texture define the proportion of light at each color channel that is not attenuated by the surface transmission.
+The values are multiplied by the `diffuseTransmissionColorFactor` to determine the total diffuse transmission color.
+```
+diffuseTransmissionColor = diffuseTransmissionColorFactor * diffuseTransmissionColorTexture.rgb
+```
+
+<table>
+  <tr>
+    <td><img src="figures/color_tex_0.0.jpg"/></td>
+    <td><img src="figures/color_tex_0.25.jpg"/></td>
+    <td><img src="figures/color_tex_0.5.jpg"/></td>
+    <td><img src="figures/color_tex_0.75.jpg"/></td>
+    <td><img src="figures/color_tex_1.0.jpg"/></td>
+  </tr>
+  <tr>
+    <td align="center">0.0</td>
+    <td align="center">0.25</td>
+    <td align="center">0.5</td>
+    <td align="center">0.75</td>
+    <td align="center">1.0</td>
+  </tr>
+  <tr>
+    <td colspan="5" align="center">
+      <em>Single-sided plane in a symmetric light setup. <code>baseColorTexture</code> and <code>diffuseTransmissionColorTexture</code> use textures that represent the different sides of the one-dollar bill. Series shows the setup at varying values of <code>diffuseTransmissionFactor</code>.</em>
+    </td>
+  </tr>
+</table>
+
+## Implementation
+
+*This section is normative.*
+
+This extension changes the `dielectric_brdf` defined in [Appendix B](../../../../specification/2.0/README.md#appendix-b-brdf-implementation)
```

Comment:
This URL should be: https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#material-structure